decentraland/unity-explorer
# Feat: analytics js-heap size

## What does this PR change?
fix #1814

- Added js-heap information to Analytics and Debug panel.
- allowed memory simulation for the release build

## How to test the changes?

1. Launch the explorer into GenesisPlaza
2. Open /debug -> Memory 
3. Observe new data regarding js-heap
4. `/goto goerli`
5. see much more `v8 engine` 's
6. Jump back to GPlaza
6. in press MEMORY Full button and observe that "Memory Budget Threshold" become '0' and that "Total Used Memory" dropped for some MB (for example, from 8500 to 8200 MB)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
